gpii/node_modules/userListeners/test/pcscTests.js

```diff
@@ -134,6 +134,28 @@ fluid.defaults("gpii.test.userListeners.pcsc", {
     }
 });
 
+/*
+ * Get an instance of the gpii.userListeners.pcsc component.
+ */
+gpii.tests.userListener.getPcscListener = function () {
+    var userListeners = gpii.userListeners({
```

Comment:
Why did these new global functions need to get added for each test function?
Note that all of this component tree will leak between tests. I'm not sure I understand what is going on here.

Reply:
Also.... nothing calls this function?